src/services/completions.ts

```diff
@@ -1066,7 +1066,7 @@ function getExhaustiveCaseSnippets(
         }
 
         const newClauses = map(elements, element => factory.createCaseClause(element, []));
-        const newLineChar = getNewLineCharacter(options, maybeBind(host, host.getNewLine));
+        const newLineChar = getNewLineOrDefaultFromHost(host);

```

Comment:
I might be wrong but should callers from services be passing in the format preferences for all of these calls?

Reply:
Yep you're right